contrib/opencensus-ext-gevent/opencensus/ext/gevent/geventcompatibility.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def patch_opencensus(event):
+    # Switch from the default runtime context using ContextVar to one using thread
+    # locals. Needed until gevent supports ContextVar.
+    # See https://github.com/gevent/gevent/issues/1407 and
+    # https://github.com/census-instrumentation/opencensus-python/issues/628
+    import opencensus.common.runtime_context as runtime_context
```

Comment:
If you wanted to attempt some future-proofing, you could surround this with `if not gevent.monkey.is_module_patched('contextvars'):` 

I'm hoping to get some form of a monkey-patch for contextvars implemented for gevent 1.5.